packages/native/src/useScrollToTop.tsx

```diff
@@ -49,53 +49,67 @@ export default function useScrollToTop(
   const route = useRoute();
 
   React.useEffect(() => {
-    let current = navigation;
+    let tabNavigators = [];
+    let currentNavigator = navigation;
```

Comment:
these are `navigation` objects and not `navigator` so I'd change the naming

Reply:
Thanks for pushing commits with your comments! I changed the variable name and I fixes TS by declaring type explicitly. Warnings are silenced locally, can we run automated tests?